src/error.ts

```diff
@@ -1,37 +1,18 @@
 import chalk from 'chalk';
-import { SkygearErrorName, SkygearError } from '@skygear/node-client';
+import { SkygearErrorNames, SkygearError } from '@skygear/node-client';
 
 export function printError(error: any) {
   let s = '';
-
   if (error instanceof SkygearError) {
-    // skygear error from sdk
-    switch (error.name as SkygearErrorName) {
-      case 'InvalidArgument':
-        if (error.info && error.info['arguments']) {
-          s = [error.message, ...(error.info['arguments'] as string[])].join(
-            '\n'
-          );
-        }
-        break;
-      case 'PermissionDenied':
-        // maybe webhook error, try getting reason from info
-        if (error.info && error.info['errors']) {
-          s = (error.info['errors'] as { reason: string }[])
-            .map(({ reason }) => reason)
-            .join('\n');
-        }
-        break;
-      default:
-        break;
-    }
-    // fallback to message if errors is empty
-    s = s || error.message;
+    // TODO(error): Pretty-print SkygearError.
+    // For now, we leverage the fact that SkygearError is well-defined JSON and
+    // print it directly.
+    s = JSON.stringify(error, null, 2);
```

Comment:
It seems like the message would not be included in JSON?

Reply:
Fixed.